tests/unit/type/zset.tcl

```diff
@@ -963,6 +963,39 @@ start_server {tags {"zset"}} {
             assert_equal 1 [r zcard z2{t}]
         }
 
+        test "Basic ZPOP - $encoding RESP3" {
+            r hello 3
+            r del z1 z2 z3 foo
+            create_zset z1 {0 a 1 b 2 c 3 d}
+            assert_equal {a 0.0} [r zpopmin z1]
+            assert_equal {d 3.0} [r zpopmax z1]
+        }
+        r hello 2
```

Comment:
to comply with #9033, we now need to move these inside the test.
this will have the advantage that if the test is skipped (possibly because the server doesn't support HELLO command), we'll skip this line too, and the disadvantage, that if the test fails, it'll not clean up after it.
```suggestion
            r hello 2
        }
```